### PR TITLE
Add some info for https://github.com/dotnet/core-eng/issues/8189

### PR DESCRIPTION
**Pull request by MattGal**

### CODE REVIEW

File: src/Microsoft.DotNet.Helix/Sdk/Readme.md [hunk 1]
```diff
@@ -80,6 +80,47 @@ Given a local folder `$(TestFolder)` containing `runtests.cmd`, this will run `r
...
+from helix.workitemutil import request_infra_retry
+
+request_infra_retry('Optional reason string')
+
+```
+
+###### Outside python:
+
+Linux / OSX: `$HELIX_PYTHONPATH -c "from helix.workitemutil import request_infra_retry; request_infra_retry('Optional reason string')"`
+Windows: `%HELIX_PYTHONPATH% -c "from helix.workitemutil import request_infra_retry; request_infra_retry('Optional reason string')"`
```
Comment (riarenas) [inline src/Microsoft.DotNet.Helix/Sdk/Readme.md]:
```suggestion

Windows: `%HELIX_PYTHONPATH% -c "from helix.workitemutil import request_infra_retry; request_infra_retry('Optional reason string')"`
```
This is rendering as the same paragraph.

File: src/Microsoft.DotNet.Helix/Sdk/Readme.md [hunk 2]
```diff
@@ -80,6 +80,47 @@ Given a local folder `$(TestFolder)` containing `runtests.cmd`, this will run `r
...
+from helix.workitemutil import request_reboot
+
+request_reboot('Optional reason string')
+
+```
+
+###### Outside python:
+
+Linux / OSX: `$HELIX_PYTHONPATH -c "from helix.workitemutil import request_reboot; request_reboot('Optional reason string')"`
+Windows: `%HELIX_PYTHONPATH% -c "from helix.workitemutil import request_reboot; request_reboot('Optional reason string')"`
```
Comment (riarenas) [inline src/Microsoft.DotNet.Helix/Sdk/Readme.md]:
```suggestion

Windows: `%HELIX_PYTHONPATH% -c "from helix.workitemutil import request_reboot; request_reboot('Optional reason string')"`
```
Same as above

**Comment by MattGal** [comment]
All the actual checks are complete here (plus this is just adding content to a .md file) so I'm going to merge.

**Pull request merged**